src/components/Nav/Nav.js

```diff
@@ -1,8 +1,372 @@
 import React from "react";
+import { Link } from "react-router-dom";
 import "./Nav.scss";
 
 const Nav = () => {
-  return <div>Nav입니다.</div>;
+  return (
+    <nav id="Nav">
+      <div id="navWrapper">
+        <Link to="/">
+          <h1 className="logo">Threppa</h1>
+        </Link>
+        <ul className="menuMain">
+          <li className="menuTap">
+            <a href="#" className="menuColor">
+              여성
+            </a>
+            <div className="menuSub">
+              <div className="subColumn">
+                <h2>슈즈 종류</h2>
+                <ul>
+                  <li>
+                    <a href="#">
+                      <i class="fas fa-hand-point-right" />
+                      클로그
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <i class="fas fa-hand-point-right" />
+                      샌들 / 웨지
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <i class="fas fa-hand-point-right" />
+                      플립 / 슬라이드
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <i class="fas fa-hand-point-right" />
+                      플랫 / 로퍼
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <i class="fas fa-hand-point-right" />
+                      부츠
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <i class="fas fa-hand-point-right" />털 안감 겨울 클로그
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <i class="fas fa-hand-point-right" />
+                      지비츠 참
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#">
+                      <i class="fas fa-hand-point-right" />
+                      양말
+                    </a>
+                  </li>
+                </ul>
+              </div>
+              <div className="subColumn">
+                <h2>여성 컬렉션</h2>
+                <ul>
+                  <li>클래식</li>
+                  <li>타이다이</li>
+                  <li>라이트라이드</li>
+                  <li>바야밴드</li>
+                  <li>크록밴드</li>
+                </ul>
+                <div className="columnUnders">
+                  <h2 className="columnUnder">여성용 슈즈 모두 보기</h2>
+                  <h2 className="columnUnder red">세일</h2>
+                </div>
+              </div>
+              <div className="subColumn">
+                <div className="navImg">
+                  <img src="./images/NavImgs/shoes1.jpg" alt="shoes1" />
```

Comment:
이미지 src 경로는 public 폴더에 절대경로로 접근할 수 있기 때문에, `./` 형태로 시작하지 않아도 됩니다!
```suggestion
                  <img src="/images/NavImgs/shoes1.jpg" alt="shoes1" />
```